tools/meson-vcs-tag.sh

```diff
@@ -18,11 +18,16 @@ else
     #
     # If the working tree has no tags (CI builds), the first git-describe will fail
     # and we fall back to project_version-commitid instead.
+
+    c=''
```

Comment:
Is this necessary to set an empty string?

Reply:
Yes, because `c` might have been set in the environment from the outside. (In principle. This is unlikely to happen in practice.)

Reply:
Ah, indeed. Thanks.